### DISCUSSION TITLE
GitHub features that might not be well-known

### DISCUSSION THREAD

**Discussion by cm890** [community/community#143420] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Question

### Body

What are some useful GitHub features that might not be well-known but can help improve my workflow? Thank you!

**Comment by longnh238** [comment, marked as answer]
GitHub offers several under-the-radar features that can make a big difference in your workflow. 
Here are a few that you might find helpful:

1. Draft Pull Requests
You can create draft pull requests to share your work early and gather feedback without worrying about it being merged prematurely. Just select "Create Draft Pull Request" when opening a new PR.

2. Saved Replies
If you frequently answer similar questions or give the exact instructions, GitHub's Saved Replies feature lets you create reusable responses that you can quickly insert into comments.

3. GitHub Actions for Automation
GitHub Actions enables you to automate repetitive tasks like testing, building, and deploying code right within your repository. There are thousands of prebuilt workflows available, so you don’t have to start from scratch.

4. Codeowners
Using a .github/CODEOWNERS file, you can automatically assign reviewers to pull requests for specific parts of your codebase, which helps distribute review work and maintain code quality.

5. Dependency Graph and Alerts
GitHub’s Dependency Graph can help you keep track of external libraries and dependencies in your project. It also provides security alerts for vulnerable dependencies, so you can keep your project safe.

Give them a try to see how they can improve your workflow.
